backend/tests/v5/unit/models/roles.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { src } = require('../../helper/path');
+
+const Roles = require(`${src}/models/roles`);
+const db = require(`${src}/handler/db`);
+
+const testCreateTeamspaceRole = () => {
+	describe('Create teamspace role', () => {
+		test('should create a new teamspace role', async () => {
+			const expectedCommand = {
+				createRole: 'team_member',
```

Comment:
same here . `team_member` should be a constant that you require from the source code